_partners/opensourceconnections.md

```diff
@@ -0,0 +1,6 @@
+---
+name: OpenSource Connections
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/osc-logomark-color.png'
```

Comment:
hello @flaxsearch - please upload the osc-logomark-color.png to '/assets/media/partners/' as part of this pull request. thanks

Reply:
@flaxsearch - wanted to follow up on this one - please let us know

Reply:
hey Kris having some trouble adding it to that folder, could you commit the attached for me?